job-dsls/src/main/resources/job-scripts/prod_pr_jobs.jenkinsfile

```diff
@@ -0,0 +1,101 @@
+
+def productVersion, buildDate
+
+pipeline {
+    agent {
+        label 'kie-rhel7 && !master'
+    }
+
+    tools {
+        maven 'kie-maven-3.8.1'
+        jdk 'kie-jdk1.8'
+    }
+
+    environment {
+        NEXUS_NIGHTLY_REPO = 'http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/groups/rhba-main-nightly'
+        NIGHTLY_STAGING_PATH = 'http://rcm-guest.app.eng.bos.redhat.com/rcm-guest/staging'
+        MAVEN_SETTINGS_ID = 'rhba-prod-main'
+        BUILD_CHECKOUT_PATH = "${WORKSPACE}/build"
+        CONFIG_CHECKOUT_PATH = "${WORKSPACE}/config"
+    }
+
+    stages {
+        stage('Prepare workspace') {
+            steps {
+                cleanWs()
+            }
+        }
+
+        stage('Get version and date for latest successful nightly') {
+            steps {
+                script {
+                    def latestNightly = sh(returnStdout: true, script: "curl -s ${NEXUS_NIGHTLY_REPO}/org/kie/rhba/rhba-parent/maven-metadata.xml | grep '<latest>'")
```

Comment:
why not to use same approach as we do here https://github.com/kiegroup/kogito-pipelines/blob/69eed80416aaf1cddee79cfd1e8d57fbf32fda5d/Jenkinsfile.prod.nightly#L44 ? :thinking: